builtin/settingtypes.txt

```diff
@@ -31,7 +31,7 @@
 #            - default
 #            - default min max
 #   * enum:
-#            - default value1,value2,...
+#            - default value1,value2,…
```

Comment:
Please don't use this special 'row of dots' character, we use separate full stops everywhere and it's clearer.